### ISSUE TITLE
Clicked like/dislike buttons, but the counter is stuck

### ISSUE THREAD

**Issue by ParkerGit2328** [lightbeam24/CustomTube#96]
I clicked the like/dislike button, then refreshed the page to check the counter, but the counter is stuck at the same numbers.

**Comment by lightbeam24** [comment]
Is this 1.2.2? If so, something weird *does* seem to be going on with the like counter. I don't think the counter is stuck though. A video was at 46 likes, I liked it, and it went down to 45 when I refreshed. Unliking and refreshing brought it up to 46 again.

In version 1.2.2, like counts are now grabbed from the API, rather than the like button element itself. This probably has something to do with it. I'll have to look into it. Maybe I'll have to revert the change.

**Comment by ParkerGit2328** [comment]
>A video was at 46 likes, I liked it, and it went down to 45 when I refreshed. Unliking and refreshing brought it up to 46 again.
Yep, I did it.

**Comment by ParkerGit2328** [comment]
Updated the title a bit.

**Comment by GemplelyTivadar** [comment]
> or going down.

@ParkerGit2328 You mean like this? Because I got this today in a private window.

https://github.com/lightbeam24/CustomTube/assets/95102462/916fc436-df0b-475c-b077-e8d64cc6dae8

**Comment by ParkerGit2328** [comment]
No, >A video was at 46 likes, I liked it, and it went down to 45 when I refreshed. Unliking and refreshing brought it up to 46 again.
